src/main/java/fi/okm/jod/yksilo/entity/Toimenkuva.java

```diff
@@ -65,6 +67,7 @@ protected Toimenkuva() {
   public Toimenkuva(Tyopaikka tyopaikka) {
```

Comment:
I suppose, tyopaikka should not be null.

Reply:
That is true, it must not be null.